types/passport-google-oauth20/index.d.ts

```diff
@@ -41,7 +41,7 @@ export interface Profile extends passport.Profile {
     _json: any;
 }
 
-export type VerifyCallback = (err?: string | Error, user?: any, info?: any) => void;
+export type VerifyCallback = (err?: string | Error, user?: Express.User, info?: any) => void;
```

Comment:
I don't fully agree with this change that does not reflect the truth.
what you verify is the response from the oauth call and it's not yet an Express.User. What is an express.user is what is returned at the end of the VerifyCallback and can be dramatically different.
It's breaking for unnecessary reason I think